games-emulation/vecx_libretro/vecx_libretro-1.2_20180412.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A port of Vecx, a GCE Vectrex emulator to the libretro API"
+DESCRIPTION="Vecx is a GCE Vectrex emulator, a vector display-based home video \
+game console that was developed by Western Technologies/Smith Engineering. It \
+had an integrated vector monitor, was monochrome and used plastic screen \
+overlays (also emulated by Vecx) to simulate color and various static graphics \
+and decorations. This is the libretro port of Vecx."
+HOMEPAGE="http://www.valavan.net/vectrex.html"
```

Comment:
The site accepts https. Shall we make this tiny change?
```diff
-HOMEPAGE="http://www.valavan.net/vectrex.html"
+HOMEPAGE="https://www.valavan.net/vectrex.html"
```
(I can update the PR if you wish.)

Reply:
Done :)